CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+## v0.2.0 (2017-06-16)
+
+* Default listen port changed to 9216.
```

Comment:
> Why

It was `9104`, that's mysqld_exporter port. Now we have our own. https://github.com/prometheus/prometheus/wiki/Default-port-allocations

> Where

EXPOSEd port in Dockerfile and default value of `-web.listen-address` flag.

> does it matter for PMM

Yes.